app/customer_route.py

```diff
@@ -0,0 +1,102 @@
+from app.models.rental import Rental
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime
+import requests
+import json
+
+
+customer_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+
+@customer_bp.route("", methods=["GET", "POST"])
+def handle_customer_get_post_all():
+    if request.method == "GET":
+
+        customer_list = Customer.query.all()
+
+        customer_response = []
+
+        for customer in customer_list:
+
+            customer_response.append(customer.json_object())
+
+        return jsonify(customer_response), 200
+
+    elif request.method == "POST":
+
+        request_body = request.get_json()
+
+        if ("name" not in request_body) or ("postal_code" not in request_body) or ("phone" not in request_body):
```

Comment:
let's take out some of the blank lines here! It makes it a little hard to read like this:

```suggestion
    if request.method == "GET":
        customer_list = Customer.query.all()
        customer_response = []
        
        for customer in customer_list:
            customer_response.append(customer.json_object())
        return jsonify(customer_response), 200
    
    elif request.method == "POST":
        request_body = request.get_json()

        if ("name" not in request_body) or ("postal_code" not in request_body) or ("phone" not in request_body):
```